.github/workflows/pull-request.yaml

```diff
@@ -29,15 +29,12 @@ jobs:
 
   e2e-tests:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.8", "3.11"]
```

Comment:
There is always clashing with API that I suspect is due to running same requests simultaneously in two different python versions. Since we test lint and types on both ends of supported python version, I think its safe to only run e2e tests in one.